Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -257,6 +257,7 @@ def com_google_fonts_check_description_broken_links(description_html):
   from lxml import etree
   doc = description_html
   broken_links = []
+  unique_links = []
```

Comment:
could be a `set()`

Reply:
does it make any difference in this case?

Reply:
Not much, it's basically nit-picking. A set is the thing you mean, both do the same here, speed won't be an issue.